INSTALL.md

```diff
@@ -221,6 +221,12 @@ If this works, then proceed to [Installation](#installation). Otherwise, ask for
     ./install.sh ../pmd-red
     ```
 
+<details>
+<summary>Note for macOS and getting `Could not find a binutils installation!` ...</summary>
```

Comment:
These backticks aren't parsed properly within HTML tags. Use `<code></code>` instead to get inline code formatting.